### ISSUE TITLE
You don't actually use sigmod for the spherical harmonic coefficients, right?

### ISSUE THREAD

**Issue by caiyongqi** [dcharatan/pixelsplat#53]
https://github.com/dcharatan/pixelsplat/blob/f4ae9c1396ce62097b407e819321f5819d7788a9/src/model/encoder/common/gaussian_adapter.py#L74

**Comment by dcharatan** [comment]
That's correct—that comment was left over from before we added support for spherical harmonics. Since we're using spherical harmonics now, the values shouldn't be mapped to the range `[0, 1]`, and so there's no sigmoid anymore. Thanks for pointing this out!
